README.md

```diff
@@ -47,17 +53,6 @@ To make it easier for developers to write record processors in other languages,
 * Handling completed and blocked tasks better during graceful shutdown
   * [PR #640](https://github.com/awslabs/amazon-kinesis-client/pull/640)
 
-#### Release 1.12.0 (October 17, 2019)
```

Comment:
Did we mean to remove this?

Reply:
Yes, removing old release notes and only keep them in changelog